ga4gh/datamodel/variants.py

```diff
@@ -903,7 +917,7 @@ def convertLocationHgvsP(self, hgvsp):
 
     def addCDSLocation(self, effect, cdnaPos):
         hgvsC = effect.hgvsAnnotation.transcript
-        if hgvsC != '':
+        if hgvsC != '' and hgvsC is not None:
```

Comment:
Python provides the idiom `if hgvsC` which checks both cases, but I changed it elsewhere at your request because it wasn't verbose enough.